examples/v2-minimal/packages/contracts/mud.config.mts

```diff
@@ -15,6 +15,13 @@ export default mudConfig({
       },
       storeArgument: true,
     },
+    EventTable: {
+      name: "event",
+      schema: {
+        value: "bool",
```

Comment:
I'm a little confused by this example. A `bool` value type but the client renders as a counter. Is there another example or name we could use that might make this more clear?

Reply:
the increment system example is super contrived. the idea was that it represents that something happened (in this case with no information because literally nothing can happen)

Reply:
agree an example where some explicit data is "set" to the ephemeral table and then displayed in the client would be a bit clearer. Maybe something like an input field that forwards a string to a system, which sets the ephemeral table?